rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+The top-level *dcp* repo MUST contain **all** significant work that is important to the DCP community as a whole and require visibility and prioritization. This includes Roadmap objectives and product-wide technical requirements (e.g. “components should have logs...“). 
+
+If Institutions are using other project management applications such as Jira for internal requirements and reporting, then all DCP related issues in a Release MUST be *projected* into ZenHub for transparency and finer granularity for tracking progress.
+
+To avoid maintenance overhead due to duplicate or incomplete issues, the creation of new items in the *dcp* repo SHOULD be limited to the PM Team or a facilitator from the Architecture Team.
+
+DCP Product Backlog items are modeled as User Stories, Epics, and Spikes. 
+ 
+#### Owners
+
+A DCP owner MUST be assigned to each issue in the Product Backlog in the *dcp* repo to coordinate progress and drive the issue to on-time closure. This owner is responsible for ensuring that related issues are created in component repo(s) and linked to Epics within the *dcp* repo.
+
+For example, an implementation task for a specific component resides in its own repo and is linked to the relevant Epic in the *dcp* repo.
+
+#### User Stories
+
+User stories are often in the form of simple templates such as:
+
+>As a \<type of user>
+I want to \<action>
+so that \<result>
+ 
+DCP issues SHOULD include a User Story, Acceptance Criteria ("Definition of Done"), and a Deliverable to be demonstrated during the Release Retrospective to Oversight.
+
+*As a data wrangler working on the DCP, I want to adapt the metadata schema quickly in order to respond to developments in experimental approaches and new assays.*
+
+*Acceptance Criteria*
+- *[ ] This*
+- *[ ] That*
+- *[ ] And one more thing*
+
+*Deliverable that can be demonstrated to Oversight*
+
+...
+
+### Planning and Executing a Release
+
+The DCP community members SHOULD respond to requests related to Release planning and execution in a timely manner. If there is a failure to engage and be responsive, then a gentle escalation to the DCP PM slack channel is RECOMMENDED to ensure that progress occurs and the schedule does not slip. 
+
+#### Timebox for Planning and Execution
+
+All timeboxes are noted as business days. Business days are included regardless of national holidays.
+
+Overlapping timeboxes denote concurrent work.
+
+Timeboxes with a range of days indicate that the work MAY be completed at any point 
+during the time box, but MUST be completed no later than the final day of the time box.
+
+Day 1 in the Release planning timeline begins in the third Week of the third Milestone of the current Release. For example Q4 planning begins in Q3M3W3 and continues into Q4M1. To reiterate, Planning continues into Milestone 1 of a Release as needed.
+
+* Prerequisite: Product Roadmap
+* Prerequisite: Optional Technical Objectives RFC
```

Comment:
If there's consensus among the _Technical Leads_ that they MUST prepare a quarterly technical RFC which is approved before Release planning starts, then this can be updated. I will add this question to **Unresolved Questions**.

Reply:
Addressed by https://github.com/HumanCellAtlas/dcp-community/pull/92/commits/4bb18c65af5b244460cf404465b1dbc4ee0cc65e